### PR TITLE
Include country codes without divisions

### PR DESCRIPTION
**Pull request by mrvisser**

### CODE REVIEW

**Comment by mrvisser** [comment]
@simong this needs yet more country codes (Palestine is missing). I'm going to make a list based on wikipedia.

**Comment by nicolaasmatthijs** [comment]
@mrvisser : Is this ready for review again?

**Comment by mrvisser** [comment]
It is now, it's also now dependent on https://github.com/oaeproject/3akai-ux/pull/4072

File: node_modules/oae-util/lib/validator.js [hunk 1]
```diff
@@ -325,7 +327,29 @@ Validator.prototype.isLongString = function() {
...
+ *
+ * @param  {String}     code    The ISO-3166-1 country code to check
+ * @return {Boolean}            Whether or not the code is a known ISO-3166-1 country code
+ * @api private
+ */
+var _hasCountryCode = function(code) {
+    if (!countriesByCode) {
+        // Lazy initialize the country code array so as to not form an cross-
+        // dependency on `oae-ui`
+        countriesByCode = _.chain(require('oae-ui').getIso3166CountryInfo().countries)
```
Comment (simong) [inline node_modules/oae-util/lib/validator.js]: The require statement should probably go at the top of the file

Comment (simong) [inline node_modules/oae-util/lib/validator.js]: Never mind, I just read the docs

**Pull request merged**